dprint.json

```diff
@@ -16,12 +16,13 @@
   "excludes": [
     "target/",
     "src-tauri/Cargo.toml",
-    "monero-sys/monero/"
+    "monero-sys/monero/",
+    ".git/**"
```

Comment:
sigh dprint. does it actually check files in `.git`? I've wanted to get rid of dprint for ages.

Reply:
It does 😭😭

Reply:
let's nuke it from orbit and just use rustfmt

Reply:
The thing is it's cool to have one formatter for both the rust and typescript code. Maybe we should just upgrade dprint...

Reply:
but nuking it also sounds cool hahaha